content/2018-12-18_dvanacty-sraz-matice.md

```diff
@@ -0,0 +1,85 @@
+Title: Dvanáctý sraz - matice
+Date: 2018-12-18 18:00:00
+Modified: 2018-12-18 18:00:00
+Author: Iveta Česalová
+
+<div class="text-justify">Po počátečním zmatku, kdy si Petr zapomněl svůj počítač, jsme poslední sraz roku zahájily opět rekapitulací, co kdo dělal. Doposud byla hra rozdělena na čtyři moduly – backend, frontend, game a testy. Nyní je v backendu pouze třída Robot, State a funkce pro načtení souřadnic, cesty k robotům, počtu políček a počáteční stav hry.</div>
+
+<div class="text-justify">V repozitáři přibylo několik souborů. Loading.py obsahuje funkce, které kompletně načtou mapu se všemi potřebnými komponenty a vlastnostmi a soubor Util.py zase třídy Tile a Direction. Soubor Validator.py kontroluje, zda je mapa správně načtena.</div>
```

Comment:
"...komponenty a vlastnostmi, a soubor..."

K souboru Validator.py - myslím, že nekontroluje, jestli je mapa správně načtená, ale jestli je mapa správně vytvořená. Jestli se nepřekrývají políčka, která se překrývat nemají, jestli tam některá nejsou víckrát, než mají být, jestli jsou správně orientovaná atp.

Reply:
jo to jsem myslela, vytvořena :-D